datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml

```diff
@@ -190,6 +190,9 @@
   description: |
     If `tls_verify` is disabled, security warnings are logged by the check.
     Disable those by setting `tls_ignore_warning` to true.
+
+    Note: `tls_ignore_warning` set to true is currently only reliable if used by one instance of one integration. 
```

Comment:
```suggestion
    Note: `tls_ignore_warning` set to true is currently only reliable if used by one instance per integration. 
```

Reply:
The behaviour is global across all integrations, I think it correct to say `if used by one instance of one integration`.